samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CachePlugin.ts

```diff
@@ -18,28 +21,35 @@ export const cachePlugin = (CACHE_LOCATION: string): ICachePlugin => {
                     }
                 });
             } else {
-                fs.writeFile(CACHE_LOCATION, cacheContext.tokenCache.serialize(), (err) => {
-                    if (err) {
-                        reject();
+                fs.writeFile(
+                    CACHE_LOCATION,
+                    cacheContext.tokenCache.serialize(),
+                    (err) => {
+                        if (err) {
+                            reject();
+                        }
```

Comment:
the formatting here is inconsistent with your if statement in my other comment, where you combined an if-statement like this into one line.